src/main/resources/scripts/terminate_historic_completed_tasks_script.sql

```diff
@@ -89,48 +94,44 @@ BEGIN
             FROM cft_task_db.tasks t
             WHERE t.task_id = task_id_var
               AND t.state IN ('COMPLETED', 'CANCELLED')
-              AND t.created < NOW() - INTERVAL '90 days'
+              AND t.last_updated_timestamp < NOW() - INTERVAL '90 days'
         ) THEN
             RAISE INFO 'No record found for task_id: %', task_id_var;
             CONTINUE;
         END IF;
 
         SELECT t.state INTO task_state
-                FROM cft_task_db.tasks t
-                WHERE t.task_id = task_id_var;
+        FROM cft_task_db.tasks t
+        WHERE t.task_id = task_id_var;
 
         -- Update the task state and other fields
         UPDATE cft_task_db.tasks t
         SET state = 'TERMINATED',
             termination_reason = CASE
-                WHEN task_state = 'COMPLETED' THEN 'COMPLETED'
-                WHEN task_state = 'CANCELLED' THEN 'CANCELLED'
+                WHEN task_state = 'COMPLETED' THEN 'cancelled'
+                WHEN task_state = 'CANCELLED' THEN 'completed'
```

Comment:
Did you check these values in the code/prod?

Reply:
Did you see the above comment about `deleted`?
Also you are setting `COMPLETED` to  `cancelled` and `CANCELLED` to `completed` which seems the wrong way around.